### DISCUSSION TITLE
Making read-write rootfs for development purposes

### DISCUSSION THREAD

**Discussion by tirankas** [OE4T/tegra-demo-distro#86] [General]
Hello,
I am using 32.5.0 branch, mender enabled based on the tegrademo distro.
My production image is RO, squashfs based. It works pretty well.
I am building a development image, ext4 based, and would like to make this read-write.
I didnt find how to disable the read-only. I have commented out the "read-only-rootfs" image feature and still rootfs is RO.

Any suggestion?

Thanks

**Comment by dwalkes** [comment]
I think it's happening here: https://github.com/OE4T/tegra-demo-distro/blob/b034c9293c6a7cc5fcc11cf00e6df42c1600c20b/layers/meta-tegrademo/classes/rootfs-postcommands-overrides.bbclass#L11

**Reply by ichergui** [reply]
Hey @tirankas
Please check the kernel `bootargs` as well.

**Reply by madisongh** [reply]
Looks like we should fix `rootfs-postcommands-overrides.bbclass` to check if `read-only-rootfs` is in IMAGE_FEATURES before applying the fstab change, rather than just assuming that Mender usage implies read-only rootfs.

**Reply by ichergui** [reply]
Agree @madisongh. 
I can do it and push a PR.

**Reply by madisongh** [reply]
Already taken care of. :)

I've booted a tegrademo-mender image with `read-only-rootfs` removed from IMAGE_FEATURES with #87 in place, and verified that the rootfs is read/write, as expected.

**Reply by ichergui** [reply]
> Already taken care of. :)
> 
> I've booted a tegrademo-mender image with `read-only-rootfs` removed from IMAGE_FEATURES with #87 in place, and verified that the rootfs is read/write, as expected.

Perfect.
Thanks @madisongh

**Reply by tirankas** [reply]
Thanks guys for your help again.
This was exactly what I was looking for
